user/useradm.go

```diff
@@ -51,22 +53,48 @@ type Config struct {
 	ExpirationTime int64
 }
 
+type ApiClientGetter func() apiclient.HttpRunner
+
+func simpleApiClientGetter() apiclient.HttpRunner {
+	return &apiclient.HttpApi{}
+}
+
 type UserAdm struct {
 	// JWT serialized/deserializer
-	jwtHandler jwt.JWTHandler
-	db         store.DataStore
-	config     Config
+	jwtHandler   jwt.JWTHandler
```

Comment:
mabe you could later rename `jwt.JWTHandler` to simply `jwt.Handler` ?

This should do the trick:
```
gorename -from '"github.com/mendersoftware/useradm/jwt".JWTHandler' -to Handler
gorename -from '"github.com/mendersoftware/useradm/jwt/mocks".JWTHandler' -to Handler
```